api/types.proto

```diff
@@ -624,6 +624,15 @@ message RaftConfig {
 	uint32 election_tick = 5;
 }
 
+
+// SecretsConfig specifies cluster-level secrets settings.
+message SecretsConfig {
+	// MaxSecrets specifies how many user-specified secrets there can be
+	uint32 max_secrets = 1;
+	// MaxSecretVersions specifies how many versions of a secret there can be
+	uint32 max_secret_versions = 2;
```

Comment:
That's a good point, and a great suggestion, thank you.  I was wondering where defaults were set.  That seems good - should I add the default values somewhere here?